### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by LeeTibbert**
We make documentation year correspond to imminent, Q1 2021 release.

### CODE REVIEW

**Comment by ekrich** [comment]
Thanks for the `SNAPSHOT` catch.

File: docs/conf.py [hunk 1]
```diff
@@ -63,9 +63,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.4.0-SNAPSHOT'
+version = u'0.4.0'
 # The full version, including alpha/beta/rc tags.
-release = u'0.4.0-SNAPSHOT'
+release = u'0.4.0'
```
[the remark this answers is not in the document]
Comment (LeeTibbert) [reply docs/conf.py:68]: Understood, I will revert that change.  A Single Point of Truth is essential.

Consistent with this comment, I am working on private studies to have
`testing.rst` use the `release` from `docs/conf.py` and the latter
figure out the SN version from a single point of truth (currently
nir/mumble/Versions.scala, but a more obvious place would be
better. That is why we do design & discussion.)
I believe that such changes, in an entirely different PR,
would make the job of the release manager easier.

File: docs/user/testing.rst [hunk 2]
```diff
@@ -11,8 +11,8 @@ To enable JUnit support, add the following lines to your `build.sbt` file:
 
 .. code-block:: scala
 
-    libraryDependencies += "org.scala-native" %%% "junit-runtime" % "0.4.0-SNAPSHOT"
-    addCompilerPlugin("org.scala-native" % "junit-plugin" % "0.4.0-SNAPSHOT" cross CrossVersion.full)
+    libraryDependencies += "org.scala-native" %%% "junit-runtime" % "0.4.0"
+    addCompilerPlugin("org.scala-native" % "junit-plugin" % "0.4.0" cross CrossVersion.full)
```
[the remark this answers is not in the document]
Comment (LeeTibbert) [reply docs/user/testing.rst]: Gone, reverted....

File: docs/conf.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (LeeTibbert) [reply docs/conf.py]: SN version changes reverted.

**Review by sjrd — APPROVED** [review]

**Comment by LeeTibbert** [comment]
Superseded by PR #2115, which generates the required year, removing an annual manual task.

**Pull request closed without merging**